curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/670ea2c4f42def724c77e3c7.md

```diff
@@ -0,0 +1,210 @@
+---
+id: 670ea2c4f42def724c77e3c7
+title: Step 38
+challengeType: 0
+dashedName: step-38
+---
+
+# --description--
+
+Now you need to retrieve the value of `entryDropdown.value` to get the currently selected option from the dropdown. 
+
+Since `entryDropdown.value` is in a static context (outside of an event listener), it only shows the value at the moment the code runs. This means it won't automatically update as the user interacts with the dropdown.
+
+It will capture the initial value (in this case, `"breakfast"`) and won't reflect any changes the user makes afterward.
```

Comment:
The explanation is good, but the instructions of adding the console.log is missing. Something like:

```suggestion
Now you need to retrieve the value of `entryDropdown.value` to get the currently selected option from the dropdown. 

Print `entryDropdown.value` to the console to see its value.

Since `entryDropdown.value` is in a static context (outside of an event listener), it only shows the value at the moment the code runs. This means it won't automatically update as the user interacts with the dropdown.

It will capture the initial value (in this case, `"breakfast"`) and won't reflect any changes the user makes afterward.
```